SPECS-EXTENDED/liblouis/liblouis.spec

```diff
@@ -0,0 +1,407 @@
+Summary:        Braille translation and back-translation library
+Name:           liblouis
+Version:        3.22.0
+Release:        2%{?dist}
+License:        LGPLv3+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            http://liblouis.org
+Source0:        https://github.com/%{name}/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  chrpath
+BuildRequires:  fdupes
+BuildRequires:  gcc
+BuildRequires:  help2man
+BuildRequires:  libyaml-devel
+BuildRequires:  make
+BuildRequires:  python3-devel
+BuildRequires:  texinfo
+Provides:       bundled(gnulib)
+%if %{with docs}
+BuildRequires:  texinfo-tex
+BuildRequires:  texlive-eurosym
+BuildRequires:  texlive-xetex
+%endif
+
+%description
+Liblouis is an open-source braille translator and back-translator named in
+honor of Louis Braille. It features support for computer and literary braille,
+supports contracted and uncontracted translation for many languages and has
+support for hyphenation. New languages can easily be added through tables that
+support a rule- or dictionary based approach. Liblouis also supports math
+braille (Nemeth and Marburg).
+
+Liblouis has features to support screen-reading programs. This has led to its
+use in two open-source screen readers, NVDA and Orca. It is also used in some
+commercial assistive technology applications for example by ViewPlus.
+
+Liblouis is based on the translation routines in the BRLTTY screen reader for
+Linux. It has, however, gone far beyond these routines.
+
+%package        devel
+Summary:        Development files for %{name}
+License:        LGPLv3+
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       pkgconfig
+
+%description    devel
+The %{name}-devel package contains libraries and header files for
+developing applications that use %{name}.
+
+%package        utils
+Summary:        Command-line utilities to test %{name}
+License:        GPLv3+
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    utils
+Six test programs are provided as part of the liblouis package. They
+are intended for testing liblouis and for debugging tables. None of
+them is suitable for braille transcription.
+
+%package -n python3-louis
+%{?python_provide:%python_provide python3-louis}
+Summary:        Python 3 language bindings for %{name}
+License:        LGPLv3+
+Requires:       %{name} = %{version}-%{release}
+Obsoletes:      %{name}-python3 < 2.6.2-3
+Provides:       %{name}-python3 = %{version}-%{release}
+BuildArch:      noarch
+
+%description -n python3-louis
+This package provides Python 3 language bindings for %{name}.
+
+%if %{with docs}
+%package doc
+Summary:        Documentation for %{name}
+License:        LGPLv3+
+Requires:       %{name} = %{version}-%{release}
+BuildArch:      noarch
+
+%description doc
+This package provides the documentation for liblouis.
+%endif
+
+%global __os_install_post %{nil}
+%global debug_package %{nil}
```

Comment:
Can we document the rationale for these flags?

Reply:
I realized that these two macros are not needed and removed them from the latest commit. Please take a look.